examples/eks_cluster_webshop/dns.tf

```diff
@@ -0,0 +1,11 @@
+resource "aws_route53_zone" "primary" {
```

Comment:
For this example I suggest to add some demo.yaml deployment with simple workload + pvc + ingress + tls.

Can be similar to https://docs.cast.ai/guides/ingress/#deployment

Reply:
Added example in README.md